packages/ui/src/components/ScrollBox.svelte

```diff
@@ -14,13 +14,27 @@
 -->
 
 <script lang="ts">
+  import { afterUpdate } from "svelte"
+
   export let vertical: boolean = false
   export let stretch: boolean = false
   export let bothScroll: boolean = false
   export let noShift: boolean = false
+  export let autoscrollable: boolean = false
+
+  let div: HTMLElement
+  let autoscroll: boolean = true
+
+  afterUpdate(async () => {
+    if (autoscrollable && autoscroll) div.scrollTo(0, div.scrollHeight)
+  })
+
+  function setAutoscroll () {
+    autoscroll = div.scrollTop > div.scrollHeight - div.clientHeight - 50
```

Comment:
why - 50?

Reply:
Threshold, must be > 20, 50 works fine, I can reduce to 30, as example, if you think 50 so much

Reply:
I'm fine, just curious.